channel/player.go

```diff
@@ -389,13 +390,35 @@ func (d *player) addEquip(item item) {
 
 func (d *player) setMesos(amount int32) {
 	d.mesos = amount
-	d.send(packetPlayerStatChange(false, constant.MesosID, amount))
+	d.send(packetPlayerStatChange(true, constant.MesosID, amount))
+	d.saveMesos()
```

Comment:
I wonder if in the future db write operations should be sent to a dedicated thread to prevent locking up the main thread?